Doc/Tutorial/chapter_cookbook.tex

```diff
@@ -1662,9 +1654,9 @@ \subsection{Creating your own substitution matrix from an alignment}
 
 %cont-doctest
 \begin{minted}{pycon}
->>> replace_info = summary_align.replacement_dictionary(["G", "A", "V", "L", "I",
-...                                                      "M", "P", "F", "W", "S",
-...                                                      "T", "N", "Q", "Y", "C"])
+>>> replace_info = summary_align.replacement_dictionary(
+...     ["G", "A", "V", "L", "I", "M", "P", "F",
+...      "W", "S", "T", "N", "Q", "Y", "C", "-"])
 \end{minted}
```

Comment:
It works for this example, but I am not sure if this works in general. Somehow `replacement_dictionary` needs to know which letters to include in the substitution matrix. If some letter of interest does not appear in the alignment anywhere, then I guess it will be missing from `replace_info`.

Reply:
This argument is characters to skip, the loss of the gapped alphabet object means we have to explicitly ignore the gap - see ffc069f881f5fb3daf5582242973ca1c7ee3dd4b

Maybe use the argument name, ``skip_chars=...`` for clarity?